include/motor-control/core/motor_driver.hpp

```diff
@@ -29,12 +29,18 @@ class MotorDriver {
     }
 
     auto read(DriverRegisters::Addresses motor_reg, uint32_t command_data)
```

Comment:
If you make this `[[nodiscard]]` it should prevent problems like this in the future